poc_mobile_verifier/src/ingest.rs

```diff
@@ -14,13 +14,9 @@ pub async fn ingest_heartbeats(
 ) -> file_store::Result<impl Stream<Item = CellHeartbeat>> {
     Ok(file_store
         .source(
-            stream::iter(
-                file_store
-                    .list_all(FileType::CellHeartbeatIngestReport, epoch.start, epoch.end)
-                    .await?,
-            )
-            .map(Ok)
-            .boxed(),
+            file_store
```

Comment:
no longer streaming these results?

Reply:
file_store.list returns a FileInfoStream, so before we were turning a stream into a Vec to turn it back into a stream.